src/store.cc

```diff
@@ -387,7 +387,7 @@ StoreEntry::destroyMemObject()
 void
 destroyStoreEntry(void *data)
 {
-    debugs(20, 3, HERE << "destroyStoreEntry: destroying " <<  data);
+    debugs(20, 3, "destroyStoreEntry: destroying " <<  data);
```

Comment:
Agreed, but this PR automatically removes HERE, and nothing else should be required. Other changes are already discussed in other comments so I will not repeat those arguments here.